src/commands/datacenters.test.js

```diff
@@ -0,0 +1,80 @@
+const should = require(`should`);
+const sinon = require(`sinon`);
+require(`should-sinon`);
+
+const util = require(`./../util`);
+const testSubject = require(`./datacenters`);
+
+// Scope vars that we’ll redefine on each test.
+let options, datacentersStub, Fastly;
+
+// Begin testing.
+describe(`datacenters`, () => {
+  // Before and after each test, we reset some stuff
+  // to ensure a fresh set of arguments. See .beforeEach()
+  // and .afterEach() near the bottom of this file.
+
+  it(`should stop without an API key`, (done) => {
+    delete options.apikey;
+    testSubject(options, Fastly, util);
+    util.apiKeyPresent.should.be.calledOnce();
+    Fastly.should.not.be.called();
+    done();
+  });
+
+  it(`should not stop without a Service ID`, (done) => {
```

Comment:
`serviceid` isn't used at all in `datacenters()`, so testing that the function does its job without this piece of data doesn't increase our confidence in the code. I would remove this whole test case.